### PR TITLE
Dev marc val

### PR DESCRIPTION
**Pull request by marcpog**

### CODE REVIEW

File: rocket_twin/tuto_marc_val/ball_to_space.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+from cosapp.base import System
+from cosapp.drivers import RungeKutta
+from cosapp.recorders import DataFrameRecorder
+import numpy as np
+import matplotlib.pyplot as plt
+
+class Ball(System):
+
```
Comment (GuyDSP) [inline rocket_twin/tuto_marc_val/ball_to_space.py:8]: why did you change this file ?

File: rocket_twin/tuto_marc_val/pipe_flow.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from cosapp.base import System, Port
+from cosapp.drivers import RungeKutta
+from cosapp.recorders import DataFrameRecorder
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+class connections_pipes(Port):
```
Comment (GuyDSP) [inline rocket_twin/tuto_marc_val/pipe_flow.py]: class name should be with Capital letter

File: rocket_twin/tuto_marc_val/pipe_flow.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    def setup(self):
+        self.add_variable('P', desc = 'pressure value', unit = 'Pa')
+        self.add_variable('Q', desc='mass flow rate', unit = 'kg/s')
+        self.add_variable('k', desc='loss coefficient', unit = 'N*s/(kg*m**2)')
```
Comment (GuyDSP) [inline rocket_twin/tuto_marc_val/pipe_flow.py]: not in the port

**Review by GuyDSP — CHANGES_REQUESTED** [review]
could you check my comments ?

File: rocket_twin/tuto_marc_val/ball_to_space.py (hunk 1 above)
Comment (marcpog) [reply rocket_twin/tuto_marc_val/ball_to_space.py:8]: Just because we needed the numpy library to generate array I think

File: rocket_twin/tuto_marc_val/pipe_flow.py (hunk 2 above)
Comment (marcpog) [reply rocket_twin/tuto_marc_val/pipe_flow.py]: ok

File: rocket_twin/tuto_marc_val/pipe_flow.py (hunk 3 above)
Comment (marcpog) [reply rocket_twin/tuto_marc_val/pipe_flow.py]: ok
